automated-comments/go/hamming/rune_or_byte_to_string_conversion.md

```diff
@@ -0,0 +1,3 @@
+As you know, using `range` over a `string` gives you a `rune`, but indexing into a `string` returns a `byte` value. The difference is described in more detail here: [Strings, bytes, runes and characters in Go](https://blog.golang.org/strings).
+
+There is no need to convert runes and bytes to string to compare them. You should decide if you want to compare `runes` or `bytes`.
```

Comment:
As a general rule, I would avoid saying "as you know" since we can't know what they know.

To tighten up:

> For consistency, comparing either `runes` or `bytes`, but not both. Ranging over a `string` gives `rune` values, whereas indexing into a `string` gives `byte` values. The difference is described in detail in [Strings, bytes, runes and characters in Go](https://blog.golang.org/strings).

Reply:
> For consistency, comparing either runes or bytes, but not both.

This sound grammatically strange to me. Shouldn't it be `compare` instead of `comparing`?